config.schema.json

```diff
@@ -186,6 +186,31 @@
       },
       "type" : "object"
     },
+    "preparationBatchingStrategy" : {
+      "description" : "Defines the batch size for target preparation. If nil, defaults to preparing 1 target at a time.",
+      "markdownDescription" : "Defines the batch size for target preparation. If nil, defaults to preparing 1 target at a time.",
+      "oneOf" : [
+        {
+          "description" : "Prepare a fixed number of targets in a single batch. `batchSize`: The number of targets to prepare in each batch.",
```

Comment:
Shouldn’t we get `markdownEnumDescriptions` in here as well? I suspect that it’s just something stupid that’s stopping it from appearing.